themes/digital.gov/src/js/gh-edit-commit.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * Sets the most recent commit timestamp for the page
+ * This is displayed in #page-data section at the bottom of the page
+ */
+
+// eslint-disable-next-line func-names
+(function () {
```

Comment:
A better way to avoid the eslint error is to use an arrow function.

```suggestion
(() => {
```